_inc/client/apps/style.scss

```diff
@@ -160,50 +160,55 @@
 	}
 }
 
-.jp-landing-apps__feature-col {
+.jp-landing-apps__feature {
 
 	@include breakpoint( ">660px" ) {
-		flex-basis: 50%;
 
-		@include breakpoint( ">660px" ) {
```

Comment:
@MichaelArestad I'd be lying if I said I intentionally fixed that. But it's been cleaned up, none the less. hahaha